<html>
<head>
  <title>VSM - Discussion</title>
  <link rel="stylesheet" href="css.css">
  <link rel="dns-prefetch" href="//fonts.googleapis.com">
  <link rel="stylesheet" id="google_fonts-css" href="//fonts.googleapis.com/css?family=Noto+Sans%3A400%2C700%7CDroid+Sans%7CPT+Sans+Narrow&#038;ver=4.7.4" type="text/css" media="all">
  <link rel="stylesheet" id="fp-font-awesome-css" href="https://28oa9i1t08037ue3m1l0i861-wpengine.netdna-ssl.com/wp-content/themes/waitbutwhy/css/fonts/font-awesome/css/font-awesome.min.css?ver=4.7.4" type="text/css" media="all">

  <meta name="keywords"    content="simplified knowledge representation, curation, ontologies, semantic web"/>
  <meta name="description" content="A simple curation tool with high flexibility."/>

  <meta itemprop="name" content="How to Easily Plant Complex Ideas into Computers">
  <meta itemprop="description" content="VSM: Intuitive representation of knowledge in computable form.">
  <meta itemprop="image" content="https://vsm.github.io/vsm-pages/img/FB-2.png">

  <meta name="twitter:card" content="summary_large_image">
  <meta name="twitter:site" content="@stcruy">
  <meta name="twitter:title" content="How to Easily Plant Complex Ideas into Computers">
  <meta name="twitter:description" content="VSM: Intuitive representation of knowledge in computable form.">
  <meta name="twitter:creator" content="@stcruy">
  <meta name="twitter:image:src" content="https://vsm.github.io/vsm-pages/img/TW-2.png">

  <meta property="og:title" content="How to Easily Plant Complex Ideas into Computers" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="https://vsm.github.io/vsm-pages" />
  <meta property="og:image" content="https://vsm.github.io/vsm-pages/img/FB-2.png" />
  <meta property="og:description" content="VSM: Intuitive representation of knowledge in computable form." />
  <meta property="og:site_name" content="The VSM-pages" />
  <meta property="fb:admins" content="404986372864030" />

  <script async src="https://www.googletagmanager.com/gtag/js?id=UA-2886799-5"></script>
  <script>
    window.dataLayer = window.dataLayer || [];
    function gtag(){dataLayer.push(arguments);}
    gtag('js', new Date());
    gtag('config', 'UA-2886799-5');
  </script>
</head>
<body>

<div id="topnavmenu" class="topnav openvert">
  <a href="javascript:void(0);" class="icon" onclick="toggleNavbarIcon()">
    <div class="l">Menu</div>
    <div class="r">&#9776;</div>
  </a>
  <div class="topnavblock">
    <a class="_fill" href="">&nbsp;</a>
    <!--
    <div class="dropdown">
      <a id="intro"  href="intro" class="dropbtn">Introduction</a> <!--href="javascript:void(0)"- ->
      <div class="dropcont">
        <a href="intro#fun-intro">General Intro</a>
        <a href="intro#deeper-intro">Deeper Intro</a>
      </div>
    </div>
    -->
    <a id="intro"      href="intro">Introduction</a>
    <a id="vsm"        href="vsm">How VSM works</a>
    <a id="summary"    href="summary">Summary</a>
    <a id="examples"   href="examples">Examples</a>
    <a id="discussion" href="discussion">Discussion</a>
    <a id="vsmgraph"   href="vsmgraph">Graphs</a>
    <a id="scicura"    href="scicura">SciCura</a>
    <a id="about"      href="about">About</a>
    <a class="_logo"   href="https://vsm.github.io"><img src="img/vsm-logo.png" height="35"></a>
  </div>
</div>
<img id="infv" style="position:absolute;top:0;left:0;" width="1" height="1">
<script>
var tnbiThrottle = 0;
function toggleNavbarIcon() {
  if(!tnbiThrottle) {
    tnbiThrottle = 1;
    var x = document.getElementById('topnavmenu');
    if(x.className == 'topnav')  x.className += ' openvert';
    else  x.className = 'topnav';
    setTimeout(function() { tnbiThrottle = 0;},  200);
  }
}
function setNavbarAActive(id) {
  document.getElementById(id).className += ' active';
  if ((window.location + '').includes('https://vsm.github.io')) {
    var u = 'https://scicura.org';
    document.getElementById('infv').src = u + '/infv/' + id;
  }
}
</script>
<script>setNavbarAActive('discussion');</script>

<div id="content" class="content">
  <h2>5. &nbsp; Discussion</h2>
<h3>i. Novelty</h3>
<p><span class="caps">VSM</span> is an entirely new and simplified approach for representing and capturing knowledge. It makes it easier for curators to make the shift from thinking in natural language to formulating in structured statements. We described how to build <span class="caps">VSM</span>-sentences: diverse statements that can contain highly contextualized information, yet in a form that is both easily readable by humans and meaningfully processable by algorithms. <span class="caps">VSM</span>-terms enable a user to disambiguate concepts with the help of ontologies and identifiers. <span class="caps">VSM</span>-connectors enable a user to disambiguate the internal structure of information, and support a uniform and intuitive method for creating statements with full structural transparency. Tridents, bidents, and lists cover many use cases; coreferences support also specialized cases. The three <span class="caps">VSM</span> Principles <a href="vsm#principle-1">&gt;</a> provide conceptual descriptions for working with terms and connectors. Templates make <span class="caps">VSM</span> as easy as filling out spreadsheets or forms, or easier because of autocomplete and CV support. Some additional features make <span class="caps">VSM</span> even more expressive.</p>
<p><span class="caps">VSM</span> thus forms a bridge between computers and human understanding. This forms the foundation for precise computational comprehension and meaningful processing of extremely diverse, human-readable quasi-sentences. It provides a solution to a long-standing problem for curation in the biosciences, and likely beyond.</p>
<p><span class="caps">VSM</span> is most closely related to: controlled languages, <span class="caps">RDF</span>, and form-based entry methods. Table 1 compares <span class="caps">VSM</span> with several knowledge representation or analysis methods, on aspects like ease-of-use and semantic internals. Figure 10 details a further comparison with <span class="caps">RDF</span>.</p>
<p><img class="i100" src="img/FigTable1.png" alt="" /></p>
<p class="caption"><b>Table 1. Comparison of <span class="caps">VSM</span> with other knowledge representation methods.</b>
– <b><span class="caps">VSM</span></b> is a knowledge representation and entry method, most closely related to controlled languages, table-based entry methods, and <span class="caps">RDF</span>. – <span class="caps">VSM</span> differs from <b>controlled languages</b>: it replaces their many fixed rules by just a few connector types, and it replaces their fixed keywords and symbols by an approach that treats all <span class="caps">VSM</span>-terms equivalently (Principle 2). <span class="caps">VSM</span> supports synonymous word forms for readability, it places all expressed meaning into terms alone, and it uses elementary connector types that enable structural consistency (Fig. 10). – <b>Table-based</b> entry methods are easy to use but poor on inherent semantics. <span class="caps">VSM</span>-templates copy their ease-of-use, but provide seamless extensibility, immediate clarity of how terms relate, and ontology-ID lookup. – <b><span class="caps">RDF</span></b> (Resource Description Framework) achieves high expressivity through triples which are also present in <span class="caps">VSM</span>. But while <span class="caps">RDF</span> is designed for IT experts, <span class="caps">VSM</span> is designed with a focus on usability for other scientific domain-experts as well: the biologist, chemist, etc. curator. Next to triples, <span class="caps">VSM</span> includes other basic units for grouping terms, and adds a view of semantics where terms are embedded in an own, local context (Principle 3). Both are essential for representing larger conceptual ideas as struc¬tured units of information, with both ease and clarity. – <span class="caps">VSM</span> is much less related to <b>natural language tagging or analysis</b> tools, despite its ability for resemblance to natural language (Principle 1). In particular, <span class="caps">VSM</span> should not be confused with text annotation (the tagging of entities and relations in free-text sentences, as how they appear in a paper), and <span class="caps">VSM</span> should not be confused with parse trees (generated by text-mining algorithms after top-down syntactic analysis of natural language; in contrast to bottom-up semantic knowledge construction with <span class="caps">VSM</span>).</p>
<hr />
<p><a name="fig-vsm-rdf"></a></p>
<p><img class="i75" src="img/Fig10.png" alt="" /></p>
<p class="caption"><b>Figure 10. Example conversion from <span class="caps">VSM</span> to <span class="caps">RDF</span>: a <span class="caps">VSM</span>-sentence is easier to understand than <span class="caps">RDF</span>.</b>
The <span class="caps">VSM</span>-sentence in (a) was manually converted to (b):&nbsp;a&nbsp;possible graph representation in <span class="caps">RDF</span>, using an example controlled-vocabulary &#8220;ex&#8221;. The relation &#8220;activates&#8221; is used as the subject of another triple and thus needs e.g. a <em>reification</em> construction in <span class="caps">RDF</span>. Here, both representations show the two distinct &#8220;device&#8221; concepts: one only located on the Moon and being activated but not yet exploding, the other one exploding as well. The bident-part &#8220;it explodes&#8221; needs to be expanded into a triple in <span class="caps">RDF</span>. The relation &#8220;ex:causes&#8221; could (as a choice) either connect the nodes &#8220;ex:activates&#8221; and &#8220;ex:belongs_to_class&#8221;, or connect their reified <em>blank nodes</em>. Note that <span class="caps">RDF</span> mixes together the supporting or <em>primitive</em> relations (&#8220;rdf:…&#8221;) and many (but not consistently all) relations that are part of the actual or expressed information (&#8220;ex:…&#8221;); while <span class="caps">VSM</span> keeps these apart as connectors and terms resp., which is a form of elegance. For most people, the <span class="caps">VSM</span>-sentence is much easier to read than the <span class="caps">RDF</span> graph.</p>
<p><a name="roadmap"></a></p>
<h3>ii. Continued implementation&nbsp;<script>function togglePopup(e) {  e = e || window.event;  e = e.target || e.srcElement || e;  var s = e.nextSibling.className;  hidePopups(e, 1);  if(s.indexOf('showPopup') < 0)  e.nextSibling.className = (s + ' showPopup');  return false;}function hidePopups(e, sk) {  if(!sk) {    e = e || window.event;  e = e.target || e.srcElement || e;    if(e.className.indexOf('popx') >= 0) return;  }  var a = document.getElementsByClassName('showPopup');  for(var i=a.length-1; i>=0; i--)  a[i].className = a[i].className.replace('showPopup', '');}document.body.addEventListener('click', hidePopups, false);document.addEventListener('keydown', function(e) {  if((e||window.event).keyCode==27 ) hidePopups(e);});</script><span class="note fun popupHolder"><span class="popx nr" onclick="return togglePopup(this)">1</span><span class="popx popup">
 Steps that we already took include:
 Design, prototype implementation, prototype application and iterative development based on user feedback;
 Redevelopment into a production-ready web component, incl. several supporting modules;
 Communication of the ideas around <span class="caps">VSM</span>, via this text among others.
</span></span></h3>
<p>To demonstrate our ideas for an interactive <span class="caps">VSM</span> UI, we first implemented a small prototype. We used it among others for creating figures with <span class="caps">VSM</span>-sentences in this text, and for interactive examples on accompanying web pages. Since then, we already rebuilt much of this work into well-designed software modules that can be reused in any curation project. These modules have a modern design that now supports community-based codevelopment of the many more possible features, and of supporting and downstream modules.<span class="note fun popupHolder"><span class="popx nr" onclick="return togglePopup(this)">2</span><span class="popx popup">
 The code is in the <a href="https://github.com/vsm"><strong>vsm</strong></a> organization on GitHub,
 and all links to information and documentation are available at
 <a href="https://vsm.github.io"><strong>vsm.github.io</strong></a>.<br />
  <br />
  The most essential modules are:
 <a href="https://github.com/vsm/vsm-box"><strong>vsm&#8209;box</strong></a>,
 <a href="https://github.com/vsm/vsm-autocomplete">vsm&#8209;autocomplete</a>, and
 <a href="https://github.com/vsm/vsm-dictionary"><strong>vsm&#8209;dictionary</strong></a>
 <a href="https://github.com/vsm/vsm-to-rdf">/&#8209;to&#8209;rdf</a>
 <a href="https://github.com/vsm/vsm-dictionary-local">/&#8209;local</a>
 <a href="https://github.com/vsm/vsm-dictionary-cacher">/&#8209;cacher</a>.
</span></span></p>
<p>A fully usable software suite should not only provide an interface for entering <span class="caps">VSM</span>-sentences, but also provide support for e.g. storing and querying them. Storage could be done in several ways: for instance as <span class="caps">JSON</span>-based graphs, or directly into a graph database. In addition, a <span class="caps">VSM</span>-to-<span class="caps">RDF</span> mapper that automatically produces <span class="caps">RDF</span> graphs (which was now done manually in Fig. 10) will have the advantage of making <span class="caps">VSM</span>-sentences readily queryable with <acronym title="SPARQL Protocol and RDF Query Language"><span class="caps">SPARQL</span></acronym><span class="note dry popupHolder"><span class="popx nr" onclick="return togglePopup(this)">1</span><span class="popx popup">
 Prud, Eric, and Andy Seaborne. &#8220;<span class="caps">SPARQL</span> query language for <span class="caps">RDF</span>.&#8221; (2006).
</span></span><span class="note fun popupHolder"><span class="popx nr" onclick="return togglePopup(this)">3</span><span class="popx popup">
 A proposed direct conversion to <span class="caps">RDF</span> is already being developed in the module <a href="https://github.com/vsm/vsm-to-rdf">vsm-to-rdf</a>.
</span></span>.
Also, a tool for building queries that directly use <span class="caps">VSM</span>-connectors or <span class="caps">VSM</span> structures as graph patterns for querying, could help users avoid the complexity of <span class="caps">RDF</span> reifications.</p>
<h3>iii. Initial adoption: use with templates</h3>
<p><em>For curators.</em> Learning new things always poses some sort of threshold. As a first introduction to <span class="caps">VSM</span>, a curation project could start off by having new curators use only templates. As most people are familiar with spreadsheets or forms, the use of <span class="caps">VSM</span>-templates does not require a steep learning curve: the curation process uses readable statements that are essentially self-explanatory as to what belongs in which empty field, and it is comfortably augmented with ontology lookup and term auto-completion. More experienced curators who encounter the need to capture information that goes beyond a template, may gradually venture into learning more about <span class="caps">VSM</span> and add and connect extra terms and structure. In a pilot curation project, we witnessed exactly this. Early adopters also appreciated <span class="caps">VSM</span> for &#8216;making the biological information&#8217;s inherent complexity much easier to handle&#8217;.</p>
<p><em>For IT experts.</em> Templates are also a good start for being able to write queries in the standard way, which is: relying on a data-schema that specifies precisely what kind of information is available for querying. Still, as a curator captures information that exceeds the template (e.g. extra details about a protein), a query could return a term, plus its attached subgraph. This would be a <span class="caps">VSM</span>-fragment that is already conveniently human-readable. If a certain type of extension occurs often, queries could be updated to search specifically for that information too. So instead of having a data-format that dictates what curators may capture, <span class="caps">VSM</span> also enables a curator-driven process, where captured information may steer query development (and standardization) too. This prospect enables a shift of focus onto semantic preservation first, which may then guide or motivate for processing technology to catch up.</p>
<p><em>With the community.</em> Templates may also be useful to ensure that captured information is readily translatable into other data formats. For example, we have been using templates in our pilot curation project, and these are helpful to export relevant parts of <span class="caps">VSM</span>-sentences and to contribute to established projects like GO Annotations and IntAct.<span class="note dry popupHolder"><span class="popx nr" onclick="return togglePopup(this)">2</span><span class="popx popup">
 &bull; <span class="caps">GOA</span>: <a href="http://geneontology.org/page/go-annotations">geneontology.org/page/go-annotations</a>
 <br />
  &bull; IntAct: <a href="https://www.ebi.ac.uk/intact">www.ebi.ac.uk/intact</a>
</span></span>Furthermore, the user-friendliness of the interface could stimulate the design of dedicated templates for entry of knowledge into other formats like <acronym title="Biological PAthway data eXchange language"><span class="caps">BioPAX</span></acronym>, <acronym title="Systems Biology Graphical Notation"><span class="caps">SBGN</span></acronym>, <acronym title="Systems Biology Markup Language"><span class="caps">SBML</span></acronym>, or GO-<span class="caps">CAM</span>.<span class="note dry popupHolder"><span class="popx nr" onclick="return togglePopup(this)">3</span><span class="popx popup">
 &bull; BioPAX: <a href="https://en.wikipedia.org/wiki/BioPAX">Wikipedia:BioPAX</a>, <a href="http://www.biopax.org">biopax.org</a>,
 <a href="https://pubmed.ncbi.nlm.nih.gov/20829833">Demir E, et al: The BioPAX community standard for pathway data sharing (2010), Nature Biotechnology</a>;
 <br />&bull; <span class="caps">SBGN</span>: <a href="https://en.wikipedia.org/wiki/Systems_Biology_Graphical_Notation">Wikipedia:SBGN</a>, <a href="http://sbgn.org">sbgn.org</a>,
 <a href="https://pubmed.ncbi.nlm.nih.gov/19668183">Le Novère N, et al: The Systems Biology Graphical Notation (2009), Nature Biotechnology</a>;
 <br />&bull; <span class="caps">SBML</span>: <a href="https://en.wikipedia.org/wiki/SBML">Wikipedia:SBML</a>, <a href="http://sbml.org">sbml.org</a>,
 <a href="https://pubmed.ncbi.nlm.nih.gov/12611808">Hucka M, et al: The systems biology markup language (<span class="caps">SBML</span>): a medium for representation and exchange of biochemical network models (2003), Bioinformatics</a>;
 <br />&bull; GO-<span class="caps">CAM</span>: <a href="http://geneontology.org/docs/gocam-overview">gocam-overview</a>,
 <a href="https://pubmed.ncbi.nlm.nih.gov/31548717">Thomas PD, et al: Gene Ontology Causal Activity Modeling (GO-<span class="caps">CAM</span>) moves beyond GO annotations to structured descriptions of biological functions and systems (2019), Nature Genetics</a>.
</span></span>Validity checks or constraints could accompany these templates to ensure compliance. Templates may also function as easy-to-conceptualize, unifying glue between different curation platforms&#8217; particular representations.</p>
<p><em>With text-mining.</em> When natural language processing (<span class="caps">NLP</span>) algorithms extract specific information types from literature, the resulting extracted facts could be automatically entered into associated <span class="caps">VSM</span>-templates. These draft <span class="caps">VSM</span> statements may then be curator-validated, corrected and/or extended. Alternatively, manually filled templates may serve as a training set for <span class="caps">NLP</span>. As an easy-to-use curation technology, <span class="caps">VSM</span> may engage more scientists in creating training data, and so accelerate a virtuous circle of curation/<span class="caps">NLP</span> co-improvement.</p>
<h3>iv. From template-guided use toward full use of <span class="caps">VSM</span></h3>
<p>In order to move from templates to using <span class="caps">VSM</span>&#8216;s full flexibility, there are some challenges to overcome.</p>
<p><em>Relation terms.</em> The first challenge is how to deal with missing terms: while controlled vocabularies are typically incomplete, <span class="caps">VSM</span> has a particular need for terms and IDs for relation concepts, which may often not yet be available. A first solution would be to enable curators to build term lists for local use, which are then flagged as new entries that should be considered by ontology experts to be included in shared controlled vocabularies.</p>
<p><em>Paraphrasing.</em> A second challenge relates to structural flexibility: there is freedom to express the same piece of information in multiple ways, with different levels of structural detail. For example, the single term &#8220;liver-cell&#8221; is equivalent to the more structurally transparent triple &#8220;cell in liver&#8221;, and both expressions will likely be entered by curators, especially when not guided by templates. This calls for mechanisms that can map structures with equivalent meaning, in order to retrieve all variants that semantically match a particular query. Still, this process will be less complicated than finding all variants that paraphrasing allows in natural language, as <span class="caps">VSM</span> presents the knowledge already in clearly defined graph structures.</p>
<p><em>Assistance.</em> Swift and nimble curation with full <span class="caps">VSM</span> may be achieved with assistance tools. First, intelligent autocompletion may be trained, to suggest e.g. common prepositions after entered relations. Also, <span class="caps">NLP</span> techniques could suggest and add <span class="caps">VSM</span>-connectors. Ultimately, algorithms could suggest partial or full translations of free text (or figures) to <span class="caps">VSM</span>. This may be accelerated by compiling a set of <span class="caps">VSM</span> translations for common free-text phrases. In the long term, a <span class="caps">VSM</span>-sentence may serve as an interface to show how a machine understood a free-text sentence.</p>
<h3>v. Potential scale of use</h3>
<p>We envision that <span class="caps">VSM</span> will first be applied in new curation projects by scientists who would otherwise use spreadsheets or need to develop their own user interface. Such practical use-cases are instructive to inspire new ways forward.  Another immediate application of <span class="caps">VSM</span> is to display various kinds of computationally generated, detailed knowledge in a human-manageable way. For instance, <span class="caps">VSM</span> may present: 1) the results of a text-mining scan over a text corpus; or 2) the detailed interpretation of what an algorithm understood from text (e.g. a scientific publication, or a social media post); or 3) a readable statement about a pattern detected in data by a machine learning or other algorithm, in a semantic form that could subsequently be queried or perhaps reasoned upon. This demonstrates that we are only at the beginning of exploring how <span class="caps">VSM</span> can be applied in various use-cases, so it is important to recognize that this publication is just the first, promising step.</p>
<p>We like to point out that <span class="caps">VSM</span>&#8216;s expressiveness enables a general-purpose curation platform, rather than the current single-purpose ones. <span class="caps">VSM</span>&#8216;s ease-of use may also facilitate a general crowdsourcing approach, to cover entire biological domains or indeed the entire life sciences. Although this would require a substantial investment in software development, and also a wide community engagement, it could pave the way to a platform that allows for full digital summaries of any research paper. <span class="caps">VSM</span>-based curation thus would provide the scientific community with the tools for converting new and existing knowledge into a structured knowledge resource. The content of a paper could be presented in what resembles a wiki page, but with information that is both human- and computer-understandable. While <em>structured digital abstracts</em> have long been a dream, <span class="caps">VSM</span> provides a major step in that direction.</p>
<p><br /></p>
<p style="text-align:right"><br />Next! :&nbsp; Read about an application of <span class="caps">VSM</span> on <a class="next" href="scicura">the SciCura page<i class="arrow right"></i></a></p>
<p style="text-align:right"><br />Or first take a peek at <a class="next" href="vsmgraph">the VSMGraphs page<i class="arrow right"></i></a></p>
<p style="text-align:right"><br />Or skip to <a class="next" href="about">the About page<i class="arrow right"></i></a></p>
<p style="text-align:left" class="prev"><br /><br /><i class="arrow left"></i>or go back to <a href="vsm">How <span class="caps">VSM</span> Works</a> or the <a href="summary">Summary</a> or the <a href="examples">Examples</a> page</p>
<div class="forPrint"> <br /><br /> <ol class="noteList fun">
<br><hr><p>Blue notes list:</p>
<li class="item"><span class="nr">1</span>
 Steps that we already took include:
 Design, prototype implementation, prototype application and iterative development based on user feedback;
 Redevelopment into a production-ready web component, incl. several supporting modules;
 Communication of the ideas around <span class="caps">VSM</span>, via this text among others.
</li>
<li class="item"><span class="nr">2</span>
 The code is in the <a href="https://github.com/vsm"><strong>vsm</strong></a> organization on GitHub,
 and all links to information and documentation are available at
 <a href="https://vsm.github.io"><strong>vsm.github.io</strong></a>.<br>The most essential modules are:
 <a href="https://github.com/vsm/vsm-box"><strong>vsm&#8209;box</strong></a>,
 <a href="https://github.com/vsm/vsm-autocomplete">vsm&#8209;autocomplete</a>, and
 <a href="https://github.com/vsm/vsm-dictionary"><strong>vsm&#8209;dictionary</strong></a>
 <a href="https://github.com/vsm/vsm-to-rdf">/&#8209;to&#8209;rdf</a>
 <a href="https://github.com/vsm/vsm-dictionary-local">/&#8209;local</a>
 <a href="https://github.com/vsm/vsm-dictionary-cacher">/&#8209;cacher</a>.
</li>
<li class="item"><span class="nr">3</span>
 A proposed direct conversion to <span class="caps">RDF</span> is already being developed in the module <a href="https://github.com/vsm/vsm-to-rdf">vsm-to-rdf</a>.
</li>

</ol> <ol class="noteList dry">
<br><hr><p>Grey notes list:</p>
<li class="item"><span class="nr">1</span>
 Prud, Eric, and Andy Seaborne. &#8220;<span class="caps">SPARQL</span> query language for <span class="caps">RDF</span>.&#8221; (2006).
</li>
<li class="item"><span class="nr">2</span>
 &bull; <span class="caps">GOA</span>: <a href="http://geneontology.org/page/go-annotations">geneontology.org/page/go-annotations</a>
 <br>&bull; IntAct: <a href="https://www.ebi.ac.uk/intact">www.ebi.ac.uk/intact</a>
</li>
<li class="item"><span class="nr">3</span>
 &bull; BioPAX: <a href="https://en.wikipedia.org/wiki/BioPAX">Wikipedia:BioPAX</a>, <a href="http://www.biopax.org">biopax.org</a>,
 <a href="https://pubmed.ncbi.nlm.nih.gov/20829833">Demir E, et al: The BioPAX community standard for pathway data sharing (2010), Nature Biotechnology</a>;
 <br>&bull; <span class="caps">SBGN</span>: <a href="https://en.wikipedia.org/wiki/Systems_Biology_Graphical_Notation">Wikipedia:SBGN</a>, <a href="http://sbgn.org">sbgn.org</a>,
 <a href="https://pubmed.ncbi.nlm.nih.gov/19668183">Le Novère N, et al: The Systems Biology Graphical Notation (2009), Nature Biotechnology</a>;
 <br>&bull; <span class="caps">SBML</span>: <a href="https://en.wikipedia.org/wiki/SBML">Wikipedia:SBML</a>, <a href="http://sbml.org">sbml.org</a>,
 <a href="https://pubmed.ncbi.nlm.nih.gov/12611808">Hucka M, et al: The systems biology markup language (<span class="caps">SBML</span>): a medium for representation and exchange of biochemical network models (2003), Bioinformatics</a>;
 <br>&bull; GO-<span class="caps">CAM</span>: <a href="http://geneontology.org/docs/gocam-overview">gocam-overview</a>,
 <a href="https://pubmed.ncbi.nlm.nih.gov/31548717">Thomas PD, et al: Gene Ontology Causal Activity Modeling (GO-<span class="caps">CAM</span>) moves beyond GO annotations to structured descriptions of biological functions and systems (2019), Nature Genetics</a>.
</li>

</ol>
</div>
</div>


<br><br><br><br><br><br><br>

<div class="footer">Site by &nbsp;<a href="about">Steven Vercruysse &amp; al.</a></div>


</body>
</html>
